scripts/base/protocols/ldap/main.zeek

```diff
@@ -0,0 +1,365 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+@load base/protocols/conn/removal-hooks
+
+@load ./consts
+
+module LDAP;
+
+export {
+  redef enum Log::ID += { LDAP_LOG, LDAP_SEARCH_LOG };
+
+  ## TCP ports which should be considered for analysis.
+  const ports_tcp = { 389/tcp, 3268/tcp } &redef;
+
+  ## UDP ports which should be considered for analysis.
+  const ports_udp = { 389/udp } &redef;
+
+  redef likely_server_ports += { LDAP::ports_tcp, LDAP::ports_udp };
+
+  ## Whether clear text passwords are captured or not.
+  option default_capture_password = F;
+
+  ## Whether to log LDAP search attributes or not.
+  option default_log_search_attributes = F;
+
+  ## Default logging policy hook for LDAP_LOG.
+  global log_policy: Log::PolicyHook;
+
+  ## Default logging policy hook for LDAP_SEARCH_LOG.
+  global log_policy_search: Log::PolicyHook;
+
+  ## LDAP finalization hook.
+  global finalize_ldap: Conn::RemovalHook;
+
+  #############################################################################
+  # This is the format of ldap.log (ldap operations minus search-related)
+  # Each line represents a unique connection+message_id (requests/responses)
+  type MessageInfo: record {
+    # Timestamp for when the event happened.
+    ts: time &log;
+
+    # Unique ID for the connection.
+    uid: string &log;
+
+    # The connection's 4-tuple of endpoint addresses/ports.
+    id: conn_id &log;
+
+    # Message ID
+    message_id: int &log &optional;
+
+    # LDAP version
+    version: int &log &optional;
+
+    # normalized operations (e.g., bind_request and bind_response to "bind")
+    opcode: set[string] &log &optional;
```

Comment:
Oh, you did the pluralization for the other ones. Did you skip this one purposely?